src/app/pages/fund/components/fiat-providers-list.tsx

```diff
@@ -35,12 +37,13 @@ export function FiatProvidersList(props: FiatProvidersProps) {
 
   return (
     <Grid
-      columnGap="auto"
+      columnGap="1.5rem"
       justifyContent="center"
       mt={['base', '48px']}
       p="0 3rem"
       rowGap="1.5rem"
-      templateColumns="repeat(auto-fill, minmax(270px, 1fr))"
+      placeItems="center"
+      gridTemplateColumns={isAtleastBreakpointMd ? 'repeat(4, 1fr)' : 'repeat(1, 1fr)'}
```

Comment:
I tried to set `gridTemplateColumns` with array syntax but I couldn't get it working properly. 

I thought it should work using this:
`gridTemplateColumns={['repeat(1, 1fr), 'repeat(4, 1fr)']`

I'm also not 100% on how our breakpoints work. Should I be able to use more than two in an array? Ideally here I would have 3 so we change from 1 to 2 to 4 columns

Reply:
I believe this should work nice 🤔
```      
gridTemplateColumns={[
        'repeat(1, 1fr)',
        'repeat(1, 1fr)',
        'repeat(2, 1fr)',
        'repeat(3, 1fr)',
        'repeat(4, 1fr)',
      ]}
```

Reply:
https://github.com/leather-wallet/extension/assets/46521087/8f03fa76-6485-4b4c-b0d1-b3d069a15ff9

Reply:
Thanks for the help, I appreciate it! I had tried that but couldn't get it working. I'll try again 👍 

It could be as I am using the new grid from `leather-styles`. If that's not working so well we can revert or else code our own using normal CSS

Reply:
I've used it in your branch 🤔

Reply:
OK thanks, it must have been just me making a mistake 👍

Reply:
Updated it with your code, thanks!